ui/components/MesheryApplications/ApplicationsCard.js

```diff
@@ -139,18 +139,22 @@ function MesheryApplicationCard({
               <Typography variant="h6" className={classes.yamlDialogTitleText}>
                 {name}
               </Typography>
-              <IconButton
-                onClick={(ev) =>
-                  genericClickHandler(ev, () => {
-                    {
-                      toggleFullScreen()
-                    }
-                  })
-                }
-                className={classes.maximizeButton}
+              <Tooltip
+                title="Fullscreen" arrow interactive placement="top"
```

Comment:
I don't think you've put into account when the card is maximised, you'll have to use a conditional operator here to change the title to "Exit Fullscreen" when the card is maximised.

Reply:
I did put it into consideration but I thought since the "fullscreen" button triggers another card(`<YAMLDialog />` at line 54 of ui/components/MesheryApplications/ApplicationsCard.js) with its own "exit fullscreen" button,  it won't be affected. 
![Screenshot 2022-09-18 at 19 32 18](https://user-images.githubusercontent.com/54249145/190922886-7e5a5b23-544b-4a86-9ea6-3897ae8f4796.png)


What do you think, should I go ahead with adding the condition?

Reply:
@YashKamboj, @pk-218, thoughts?

Reply:
@warunicorn19 as @dara-soji mentioned, the 'exit fullscreen' button appears on the `YAMLDialog` component and there is no need to add the condition.

Reply:
And I see there is [no ToolTip](https://github.com/meshery/meshery/blob/master/ui/components/YamlDialog.js#L27) to the exit fullscreen button on the `YAMLDialog` component.

How about we have a ToolTip for that as well @dara-soji as a part of this PR?

Reply:
Will gladly update the PR with the TollTip in the YAMLDialog component. 
Thanks for the feedback.